gensim/models/wrappers/fasttext.py

```diff
@@ -337,8 +346,12 @@ def init_ngrams(self):
         """
         self.wv.ngrams = {}
         all_ngrams = []
+        self.wv.syn0 = np.zeros((len(self.wv.vocab), self.vector_size), dtype=REAL)
+
         for w, v in self.wv.vocab.items():
             all_ngrams += self.compute_ngrams(w, self.wv.min_n, self.wv.max_n)
+            self.wv.syn0[self.wv.vocab[w].index] += np.array(self.wv.syn0_all[self.wv.vocab[w].index])
```

Comment:
`v` instead of `self.wv.vocab[w]`? Please rename the variable to something more verbose when making this change.